probe-rs/src/vendor/espressif/sequences/esp32.rs

```diff
@@ -178,4 +193,12 @@ impl XtensaDebugSequence for ESP32 {
 
         Ok(())
     }
+
+    fn on_unknown_semihosting_command(
+        &self,
+        interface: &mut Xtensa,
+        details: UnknownCommandDetails,
+    ) -> Result<bool, crate::Error> {
+        EspBreakpointHandler::handle_idf_semihosting(interface, details)
```

Comment:
From my memory, shouldn't only the riscv esp's emit semihosting calls to indicate that it wants the debugger to set a breakpoint? Not talking about the other stuff that might be communicated over semihosting. E.g where do you see that xtensa esp's need to handle semihosting calls on breakpoints?

Reply:
Thanks, fair. Removed from Xtensa although it would just be unused.